### DISCUSSION TITLE
how to create pads in the test files ?

### DISCUSSION THREAD

**Discussion by OmarLOay99** [The-OpenROAD-Project/OpenROAD#1949] [Q&A]
When I run any of the test files such as gcd_ sky130hd.tcl file on the GUI, I realized that there are no Pad cells in the layout. So I am asking if there is a way to source the Pads within the TCL file.

**Comment by vijayank88** [comment]
@OmarLOay99 
Are you looking for the test case with pads?

If yes then check this: https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/pad/test
